ui/src/pages/Dashboard/DataIngestion.tsx

```diff
@@ -102,43 +109,54 @@ export const DataIngestionDashboard = ({
         debounceHandlerChangeData(value, toggleNoTraffic);
     };
 
-    const handleChangeSwitch = (e: unknown, { value }: { value?: unknown }) => {
-        setToggleNoTraffic(!value);
-        debounceHandlerChangeData(searchInput, !value);
-    };
+    const infoMessage = VIEW_BY_INFO_MAP[viewByInput];
 
     return (
         <>
             <DashboardContextProvider
                 preset={EnterpriseViewOnlyPreset}
                 initialDefinition={dashboardDefinition}
             >
-                <DashboardCore
-                    width="100%"
-                    height="auto"
-                    dashboardCoreApiRef={setDashboardCoreApi}
-                />
+                <>
+                    <DashboardCore
+                        width="100%"
+                        height="auto"
+                        dashboardCoreApiRef={setDashboardCoreApi}
+                        actionMenus={getActionButtons('data_ingestion')}
+                    />
+
+                    <div id="data_ingestion_search" className="invisible_before_Moving">
+                        <p id="data_ingestion_search_label">Search</p>
+                        <Search
+                            id="data_ingestion_search_input"
+                            onChange={handleChangeSearch}
+                            value={searchInput}
+                            style={{ minWidth: '150px', gridRow: '6', gridColumn: '1' }}
+                        />
+                    </div>
+                    {/* <div id="switch_hide_no_traffic_wrapper" className="invisible_before_Moving">
```

Comment:
commented code?

Reply:
yes, thats on purpose and it is logic about toggle switch that should not be used currently but is planned for future. So potentially it will be just uncommented and added as next improvement. Obviously there also might be decision to completely drop it, so then we can just remove that code.